src/service/randomNumberInRange.test.js

```diff
@@ -0,0 +1,121 @@
+        const forbidden = [2, 4];
+
+        //when 
+        const randomNumber = randomNumberInRange(minimum, maximum, forbidden);
+
+        //then 
+        expect(randomNumber).toBeLessThanOrEqual(5);
+        expect(randomNumber).toBeGreaterThanOrEqual(0);
+        expect(randomNumber).not.toBe(2);
+        expect(randomNumber).not.toBe(4);
+    });
+
+    it('Should return a number from -100 to 50, but not 8', () => {
+
+        //given
+        const minimum = -100;
+        const maximum = 50;
+        const forbidden = [8];
+
+        //when 
+        const randomNumber = randomNumberInRange(minimum, maximum, forbidden);
+
+        //then 
+        expect(randomNumber).toBeLessThanOrEqual(50);
+        expect(randomNumber).toBeGreaterThanOrEqual(-100);
+        expect(randomNumber).not.toBe(8);
+    });
+
+    it('Should return a random number from 7 to 41', () => {
+
+        //given
+        const minimum = 7;
+        const maximum = 41;
+
+        //when 
+        const randomNumber = randomNumberInRange(minimum, maximum);
+
+        //then 
+        expect(randomNumber).toBeLessThanOrEqual(41);
+        expect(randomNumber).toBeGreaterThanOrEqual(7);
+    });
+
+    it('Should return an error when min is greater than max', () => {
+
+        //given
+        const minimum = 9;
+        const maximum = 3;
+        const forbidden = [8];
+
+        //when 
+        const randomNumberInvoke = () => {randomNumberInRange(minimum, maximum, forbidden);} 
+
+        //then 
+        expect(randomNumberInvoke).toThrowError('Invalid arguments!');
+    });
+
+    it('Should return an error when forbidden is not an array', () => {
+
+        //given
+        const minimum = 7;
+        const maximum = 10;
+        const forbidden = 8;
+
+        //when 
+        const randomNumberInvoke = () => {randomNumberInRange(minimum, maximum, forbidden);} 
+
+        //then 
+        expect(randomNumberInvoke).toThrowError('Invalid arguments!');
+    });
+
+    it('Should return an error when min is not a number', () => {
+
+        //given
+        const minimum = '7';
+        const maximum = 10;
+        const forbidden = [8];
+
+        //when 
+        const randomNumberInvoke = () => {randomNumberInRange(minimum, maximum, forbidden);} 
+
+        //then 
+        expect(randomNumberInvoke).toThrowError('Invalid arguments!');
+    });
+
+    it('Should return an error when max is not a number', () => {
+
+        //given
+        const minimum = 7;
+        const maximum = {max: 10};
+        const forbidden = [8];
+
+        //when 
+        const randomNumberInvoke = () => {randomNumberInRange(minimum, maximum, forbidden);} 
+
+        //then 
+        expect(randomNumberInvoke).toThrowError('Invalid arguments!');
+    });
+
+    it('Should return an error when every possible outcome is forbiddenn', () => {
+
+        //given
+        const minimum = 4;
+        const maximum = 8;
+        const forbidden = [7, 5, 6, 4, 8];
```

Comment:
[Readability]
I know that you wanted to check what happens when somebody tries to give all numbers (but shuffled).
It is not big deal, but you could use smaller example (i.e. from 0 to 2) and also make numbers "nearly sorted" like: `[8, 4, 5, 6, 7]` to make it as easy as possible to find that it indeed is `every possible outcome forbidden`